packages/react-components/react-avatar/src/components/Avatar/Avatar.types.ts

```diff
@@ -11,6 +11,11 @@ export type AvatarSizes = AvatarSize;
  */
 export type AvatarSize = 16 | 20 | 24 | 28 | 32 | 36 | 40 | 48 | 56 | 64 | 72 | 96 | 120 | 128;
 
+/**
+ * Shape of the avatar
+ */
+export type AvatarShape = 'circular' | 'square';
+
```

Comment:
Would it be possible to split the Avatar related changes into another PR?

Reply:
I created #27738 for avatar shape